modules/cluster-logging-log-store-status-viewing.adoc

```diff
@@ -173,7 +173,7 @@ status:
         type: Unschedulable
 ----
 
-This status message indicates that the log store CR uses a non-existent PVC.
+The following status message indicates that the log store CR uses a non-existent PVC.
```

Comment:
PVC: Do we need to expand this if this is the first occurrence? Please check and do the needful.

Reply:
Updated. Thanks.